scribejava-apis/src/main/java/com/github/scribejava/apis/service/Doximity20ServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.github.scribejava.apis.service;
+
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.model.AbstractRequest;
+import com.github.scribejava.core.model.OAuth2AccessToken;
+import com.github.scribejava.core.model.OAuthConfig;
+import com.github.scribejava.core.model.OAuthConstants;
+import com.github.scribejava.core.oauth.OAuth20Service;
+
+public class Doximity20ServiceImpl extends OAuth20Service {
+
+
+    /**
+     * Default constructor
+     *
+     * @param api    OAuth2.0 api information
+     * @param config OAuth 2.0 configuration param object
+     */
+    public Doximity20ServiceImpl(DefaultApi20 api, OAuthConfig config) {
+        super(api, config);
+    }
+
+    @Override
+    protected <T extends AbstractRequest> T createAccessTokenRequest(String oauthVerrifier, T request) {
+        super.createAccessTokenRequest(oauthVerrifier,request);
```

Comment:
I think you should change
`super.createAccessTokenRequest(oauthVerrifier,request);`
to
`super.createAccessTokenRequest(oauthVerrifier, request);`